docs/usage/dash-ui.rst

```diff
@@ -0,0 +1,85 @@
+.. _dash-ui:
+
+###########
+Dash Web UI
+###########
+
+Navigator for DWD open weather data.
+
+
+************
+Introduction
+************
+
+Welcome to Wetterdienst UI, your friendly web-based GUI for the Wetterdienst
+weather service library for Python. This web UI can easily be self-hosted.
+
+The implementation is still its infancy, so we are happy about further
```

Comment:
*in its infancy

Reply:
Thanks for spotting this typo. I've just corrected it.